components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/services/InputValidationManagementServiceImpl.java

```diff
@@ -125,35 +138,29 @@ public Map<String, Validator> getValidators(String tenantDomain) {
     }
 
     @Override
-    public List<ValidationConfiguration> getConfigurationFromUserStore(String tenantDomain)
+    public ValidationConfiguration getConfigurationFromUserStore(String tenantDomain, String field)
```

Comment:
Changing public method will affect the caller methods, how we are going to handle it?